extension/js/common/composer.ts

```diff
@@ -99,6 +100,10 @@ export class Composer {
   public canReadEmails: boolean;
   public initialized: Promise<void>;
 
+  public get Recipients() {
```

Comment:
What does this do? I'm not familiar with the `get` syntax.

Reply:
It's a getter, I have learned it from C#,

I can write the same thing using the method, something like this:
```TS
public getRecipients() {
  return this.addedRecipients;
}
```

but the way I've written it takes less code

Reply:
hmm... no problem with it. But it is not used anywhere else in the code, so it looks alien. If we were to start using it, we should refactor the whole codebase with such code for consistency. But maybe in this case the benefit is very marginal, so I think it's not worth it. If you don't mind, please use the more common syntax.

We will definitely one day do a big code style / consistency refactor, and can consider it at that point.

Reply:
Okay, I agree :+1: